### PR TITLE
Fix dockerfile download location for sigsci module

### PR DESCRIPTION
**Pull request by gavinmcnair**
Update download URL

### CODE REVIEW

**Review by mliu-mettle — APPROVED** [review]

**Review by swade1987 — APPROVED** [review]

**Review by benhartley — APPROVED** [review]

**Review by benhartley — APPROVED** [review]

File: Makefile [hunk 1]
```diff
@@ -1,7 +1,7 @@
 CURRENT_WORKING_DIR=$(shell pwd)
 
 NGINX_INGRESS_CONTROLLER_IMAGE := eu.gcr.io/k8s-artifacts-prod/ingress-nginx/controller
-NGINX_INGRESS_CONTROLLER_VERSION := v0.34.1
+NGINX_INGRESS_CONTROLLER_VERSION := v0.35.0
```
Comment (swade1987) [inline Makefile:4]: I am now remaining you Massimo!! 😉

**Review by swade1987 — APPROVED** [review]

File: Makefile (hunk 1 above)
Comment (gavinmcnair) [reply Makefile:4]: My wife would call you a dieb (thief/bad person) in swiss german.

**Review by paulstakem — APPROVED** [review]

**Comment by gavinmcnair** [comment]
Towards https://github.com/eeveebank/platform-engineering/issues/1436

File: Makefile (hunk 1 above)
Comment (gavinmcnair) [reply Makefile:4]: Im only joking :+1:

**Pull request merged**